src/Api/Auth/Controllers/TwoFactorController.cs

```diff
@@ -217,8 +226,16 @@ public async Task<TwoFactorDuoResponseModel> PutDuo([FromBody] UpdateTwoFactorDu
 
         try
         {
-            var duoApi = new DuoApi(model.IntegrationKey, model.SecretKey, model.Host);
-            await duoApi.JSONApiCall("GET", "/auth/v2/check");
```

Comment:
❓ was the removal of `await duoApi.JSONApiCall("GET", "/auth/v2/check");` intentional?